### DISCUSSION TITLE
SyntaxError: Unexpected token '{' on iOS (both app and browser)

### DISCUSSION THREAD

**Discussion by 0x07cf-dev** [dani-garcia/vaultwarden#5015] [Q&A] [answered]
I can access my instance from all my machines, both via browser and app, but I cannot access my vault on iOS anymore. I have opened Safari's dev tools and the console shows the error in the title.

My vaultwarden is updated (via apt on a DietPi machine) and **this only happens on my iOS phone**; macOS, Windows and Linux all work fine both via browser and their respective apps, only iOS stopped working. On iOS, the browser gets stuck on an endless loading screen and no login options show up, whereas the app simply shows a generic error when trying to login.

This is the line involved:
`(self.webpackChunk_bitwarden_web_vault = self.webpackChunk_bitwarden_web_vault || []).push([[734, 44], {`

Somehow that last '{' throws a syntax error, again, only on iOS.

Thank you very much for your time.

**Comment by BlackDex** [comment, marked as answer]
It probably goes wrong somewhere else which the parser can't detect correctly.
Try the `testing` tagged Vaultwarden image and try again.

**Reply by 0x07cf-dev** [reply]
Thank you very much for the hasty reply! It just worked out of the box.
I stopped the service, I downloaded the testing image, I then extracted the binaries and updated both server and the web vault. It just started working with the testing image, no more errors. 

I then tried with the previous non-testing server version and it worked again, as it always has. I am pretty sure this was due to an outdated web vault, though it looked weird to me that only iOS had the issue, and not all my machines.

**Comment by BlackDex** [comment]
The app doesn't use the web-vault, so that couldn't have been the issue actually. Not sure what it could have been then
